lib/Backend/LowerMDShared.cpp

```diff
@@ -4713,20 +4713,50 @@ LowererMD::GenerateLoadPolymorphicInlineCacheSlot(IR::Instr * instrLdSt, IR::Reg
 }
 
 void
-LowererMD::ChangeToWriteBarrierAssign(IR::Instr * assignInstr)
+LowererMD::ChangeToWriteBarrierAssign(IR::Instr * assignInstr, const Func* func)
 {
-    ChangeToAssign(assignInstr);
-
-    // Now insert write barrier if necessary
 #ifdef RECYCLER_WRITE_BARRIER_JIT
```

Comment:
nit: Why start a new `#ifdef` section? Can just put in existing `#ifdef` section below after `ChangeToAssign`.